app/models/bet_option.rb

```diff
@@ -1,6 +1,7 @@
 class BetOption < ApplicationRecord
   has_many :users, through: :user_bets
   has_many :user_bets
+  belongs_to :bet
```

Comment:
I am ok with this, but if you are changing the model architechture, you may want to look at the rest of the relationships involved as well.

Currently: Bet has many options through user bets, but bet option belongs to a bet directly.